proto/config/appconfig.proto

```diff
@@ -206,6 +206,15 @@ message AppInstanceConfig {
 
   // PatchEnvelopeRef is used to track all patch envelopes used by the app instance
   PatchEnvelopeRef patchRef = 23;
+
+  // This edge-node UUID for the Designate Node for the Application
+  string DesignatedNodeID = 24;
```

Comment:
For new fields it makes sense to follow the lower_snake_case. Note that the protoc compiler create golang struct members in UpperCamelCase thus the code using those definitions do not need to be changed.

Reply:
Done